app/models/qernel/merit_facade/load_shifting_adapter.rb

```diff
@@ -79,12 +79,12 @@ def limiting_curve
       end
 
       def input_capacity
-        # The input capacity will be further affected in Merit by the availability of the node.
-        @input_capacity ||= limiting_curve.max
+        @input_capacity = (@config.input_capacity_from_share || 0.0) * output_capacity
       end
 
       def output_capacity
-        input_capacity
+        # The input capacity will be further affected in Merit by the availability of the node.
```

Comment:
Is this comment still applicable here?

Either this should now be a comment about the output capacity, or it should be moved to the `input_capacity` method. Check Merit to see which one is the correct option.

Reply:
I checked the (load_shifting module)[https://github.com/quintel/merit/blob/master/lib/merit/flex/load_shifting.rb] in Merit and to me it seems like neither the input nor the output capacity is changed or calculated there.  It does seem like those are used to calculated the (future) availability however.

Should I remove the comment altogether?

Reply:
I think the confusion was mainly about the comment referring to the input capacity in the definition of the output capacity. I have updated the comments.